HacktoberfestProject.Web/Data/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using HacktoberfestProject.Web.Models;
+using HacktoberfestProject.Web.Data.Entities;
+
+namespace HacktoberfestProject.Web.Data.Repositories
+{
+	public class UserRepository : IUserRepository
+	{
+		private ITableContext _tableContext;
+
+		public UserRepository(ITableContext tableContext)
+		{
+			_tableContext = tableContext ?? throw new ArgumentNullException(nameof(tableContext));
+		}
+
+		public async Task<User> CreateAsync(User user)
+		{
+			UserEntity userEntity = (UserEntity)user;
+
+			return (User)await _tableContext.InsertOrMergeEntityAsync(userEntity);
```

Comment:
I would have preferred these `await`s to be on a new line against a variable and return the variable for greater readability. I know it adds extra line of code but it's clearer for less experienced developers